Package.swift

```diff
@@ -3,7 +3,11 @@ import PackageDescription
 let package = Package(
     name: "Jay",
     targets: [
-        Target(name: "JayExample", dependencies: ["Jay"]),
-        Target(name: "JayPerformance", dependencies: ["Jay"])
+        // Target(name: "JayExample", dependencies: ["Jay"]),
```

Comment:
You can just remove these lines, we can always pull them back from source control.